doc/quickstart/find_data.rst

```diff
@@ -564,6 +564,75 @@ explained in :ref:`extra_facets`, and which content is :download:`available here
 </../esmvalcore/config/extra_facets/ipslcm-mappings.yml>`. These multi-variable
 files must also undergo some data selection.
 
+.. _read_access-esm:
+
+ACCESS-ESM
+^^^^^^^^^^
+
+ESMValTool can read native `ACCESS-ESM <https://research.csiro.au/access/about/esm1-5/>`__
+model output.
+
+.. warning::
+
+  This is the first version of ACCESS-ESM CMORizer for ESMValCore Currently, 
+  Supported variables: ``pr``, ``ps``, ``psl``, ``rlds``, ``tas``, ``ta``, ``va``,
+  ``ua``, ``zg``, ``hus``, ``clt``, ``rsus``, ``rlus``.
+
+This is an example of the directory file structure in `config_developer.yml`.
+
+.. code-block:: yaml
+
+  ACCESS:
+    cmor_strict: false
+    input_dir:
+      default:
+        - '{institute}/{sub_dataset}/{exp}/{modeling_realm}/netCDF'
+    input_file:
+      default: '{sub_dataset}.{special_attr}-*.nc'
+    output_file: '{institute}_{sub_dataset}_{special_attr}_{short_name}'
+    cmor_type: 'CMIP6'
+    cmor_default_table_prefix: 'CMIP6_'
+
+.. hint::
+
+  We only provide one default `input_dir` since this is how ACCESS-ESM native data was 
+  stored on NCI. Users can modify this path to match their local file structure.
+
+
+Thus, example dataset entries could look like this:
+
+.. code-block:: yaml
+
+  dataset:
+    - {project: ACCESS, institute: ACCESS-ESM1-5, mip: Amon, dataset:ACCESS_ESM, sub_dataset: HI-CN-05, 
+      exp: history, modeling_realm: atm, special_attr: pa, start_year: 1986, end_year: 1986}
+
+
+`dataset` and `sub_dataset` are not redundant, `dataset` is for ESMValCore to search for CMORizer,
+which is always `ACCESS_ESM`, `sub_dataset` is dataset under root `ACCESS-ESM1-5`.
+
+`special_attr` is a special attribute in the filename `ACCESS-ESM` raw data, it's related to frquency
+of raw data.
+
+`modeling_realm` is a realm attribute, it include `atm`, `ice` and `oce`.
+
+Similar to any other fix, the ACCESS-ESM fix allows the use of :ref:`extra
+facets<extra_facets>`.
+By default, the file :download:`emac-mappings.yml
+</../esmvalcore/config/extra_facets/access-mappings.yml>` is used for that
+purpose.
+For some variables, extra facets are necessary; otherwise ESMValCore cannot
+read them properly.
+Supported keys for extra facets are:
+
+==================== ====================================== =================================
+Key                  Description                            Default value if not specified
+==================== ====================================== =================================
+``raw_name``         Variable name of the variable in the   CMOR variable name of the
+                     raw input file                         corresponding variable
+``modeling_realm``   Realm attribute include `atm`, `ice`   Default realm of this variable 
+                     and `oce`      
```

Comment:
```suggestion
``modeling_realm``   Realm attribute include `atm`, `ice`   No default (needs to be 
                     and `oce`                              specified in extra facets or
                                                            recipe if default DRS is used)
```

Technically, since you're using `NativeDatasetFix.get_cube`, `raw_units` is also allowed (see https://docs.esmvaltool.org/projects/ESMValCore/en/latest/quickstart/find_data.html#emac)